iocage/cli/create.py

```diff
@@ -53,21 +53,28 @@ def validate_count(ctx, param, value):
 
 @click.command(name="create", help="Create a jail.")
 @click.option("--count", "-c", callback=validate_count, default="1")
-@click.option("--release", "-r", required=False)
-@click.option("--template", "-t", required=False)
+@click.option("--release", "-r", required=False,
+              help="Specify the release to use for the new jail.")
+@click.option("--template", "-t", required=False,
+              help="Flag this jail as a template, which allows for rapid "
+                   "redeployment of a customized jail.")
 @click.option("--pkglist", "-p", default=None)
```

Comment:
`-p` allows you to specify a JSON file who's purpose is to install each package inside the jail after creation.

Example:
```
{
    "pkgs": [
    "foo",
    "bar"
    ]
}
``` I have an error prompt for the user if it's in an incorrect format, so perhaps just explaining the concept is enough without also showing the format of the JSON.